ui/app/src/components/PreviewAudiencesPanel/AudiencesPanelUI.tsx

```diff
@@ -30,10 +30,11 @@ import CheckboxGroup from '../FormEngineControls/CheckboxGroup';
 import DateTime from '../FormEngineControls/DateTime';
 import LookupTable from '../../models/LookupTable';
 import { Alert } from '@mui/material';
+import { create8601String, get8601Pieces } from '../../utils/datetime';
 
 interface AudiencesPanelUIProps {
   model: ContentInstance;
-  fields: LookupTable<ContentTypeField>;
+  fields: LookupTable<ContentTypeField & { timeZone?: string }>;
```

Comment:
AudiencesPanel fields happen to match the CotentTypeField type, but it doesn't have a `timeZone`. Should I create a separate type for this since the match is not exact (e.g. AudiencesPanel fields do not have validations)?

Reply:
I meant timeZone could be treated as a field.
But, is this still applicable based on the last discussions? We're not tracking the time zone value separately from the 8610 string. Can we remove this stuff?

Reply:
the thing is that timeZone is not a field itself, but a part of a field (when the control is date-time). Will explain a bit better on the latter comment.

Reply:
What about now? Is this still applicable?

Reply:
doesn't apply anymore, already removed in https://github.com/craftercms/studio-ui/pull/3399/commits/273671a2d0590d12ae6a353964732c71b499ae17